### ISSUE TITLE
모델 로딩 warning 질문

### ISSUE THREAD

**Issue by austin-watcha** [monologg/KoELECTRA#38]
## Checklist

- [Y] I've searched the project's [`issues`]

## ❓ Question

안녕하세요, 우선 좋은 프로젝트를 공유해주셔서 감사합니다.
제가 만드신 모델을 로드할 때 아래와 같은 경고 메세지가 뜨는데요

`
Some weights of the model checkpoint at monologg/koelectra-small-v2-discriminator were not used when initializing ElectraModel: ['discriminator_predictions.dense_prediction.bias', 'discriminator_predictions.dense.bias', 'discriminator_predictions.dense_prediction.weight', 'discriminator_predictions.dense.weight']
`

위 메세지는 `ElectraModel`이 아닌 `ElectraForPreTraining`을 import 하면 생기지 않는 것을 확인했고, `ElectraConfig.get_config_dict('monologg/koelectra-small-v2-discriminator')` 를 수행해서 나오는 결과도 `{architectures: ElectraForPreTraining}` 였습니다.

혹시 huggingface 없데이트에 따라 생긴 변화일까요?

`ElectraModel`로서 모델을 로드하고 받는 output은 {`logits`, output of [batch_size * sequence_len]} 이고,
`ElectraForPreTraining`로서 모델을 로드하고 받는 output은 {`last_hidden_state`, output of [batch_size * sequence_len * hidden_size]} 입니다.

저는 discirinator를 사용하고 싶은데, 위 메세지가 나오더라도 `ElectraModel`로 로드하는게 옳은 사용법 일까요?

**Comment by monologg** [comment]
discriminator까지 사용하고 싶으시면 `ElectraForPretraining`을 사용하셔야 합니다.

아래 코드 참고해주세요:)

https://github.com/huggingface/transformers/blob/05d3a43c59dadf815fe46df8fed8cc8e816c8588/src/transformers/models/electra/modeling_electra.py#L1065-L1072

https://github.com/huggingface/transformers/blob/05d3a43c59dadf815fe46df8fed8cc8e816c8588/src/transformers/models/electra/modeling_electra.py#L1137

**Comment by austin-watcha** [comment]
넵 답변 감사합니다!
